### ISSUE TITLE
Question for training process

### ISSUE THREAD

**Issue by batho2n** [NVIDIA/tacotron2#237]
I am following the readme process

I finished stepup step. (pip install -r requirements.txt)
Next, I executed training process
`$ python train.py --output_directory=outdir --log_directory=logdir`
and I got the error message
```
RuntimeError: module compiled against API version 0xc but this version of numpy is 0xb
ImportError: numpy.core.multiarray failed to import
ImportError: numpy.core.umath failed to import
ImportError: numpy.core.umath failed to import
2019-07-17 20:41:59.109699: F tensorflow/python/lib/core/bfloat16.cc:675] Check failed: PyBfloat16_Type.tp_base != nullptr
Aborted (core dumped)
```

So I executed
`$ pip install numpy --upgrade`
and I executed training process again,
```
Traceback (most recent call last):
  File "train.py", line 17, in <module>
    from hparams import create_hparams
  File "/home/ubuntu/users/batho2n/tacotron2/hparams.py", line 1, in <module>
    import tensorflow as tf
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorflow/__init__.py", line 35, in <module>
    from tensorflow._api.v1 import compat
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorflow/_api/v1/compat/__init__.py", line 22, in <module>
    from tensorflow._api.v1.compat import v2
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorflow/_api/v1/compat/v2/__init__.py", line 303, in <module>
    from tensorboard.summary._tf import summary
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/summary/__init__.py", line 25, in <module>
    from tensorboard.summary import v1
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/summary/v1.py", line 24, in <module>
    from tensorboard.plugins.audio import summary as _audio_summary
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/plugins/audio/summary.py", line 36, in <module>
    from tensorboard.plugins.audio import metadata
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/plugins/audio/metadata.py", line 21, in <module>
    from tensorboard.compat.proto import summary_pb2
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/compat/proto/summary_pb2.py", line 15, in <module>
    from tensorboard.compat.proto import tensor_pb2 as tensorboard_dot_compat_dot_proto_dot_tensor__pb2
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/compat/proto/tensor_pb2.py", line 15, in <module>
    from tensorboard.compat.proto import resource_handle_pb2 as tensorboard_dot_compat_dot_proto_dot_resource__handle__pb2
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboard/compat/proto/resource_handle_pb2.py", line 22, in <module>
    serialized_pb=_b('\n.tensorboard/compat/proto/resource_handle.proto\x12\x0btensorboard\"r\n\x13ResourceHandleProto\x12\x0e\n\x06\x64\x65vice\x18\x01 \x01(\t\x12\x11\n\tcontainer\x18\x02 \x01(\t\x12\x0c\n\x04name\x18\x03 \x01(\t\x12\x11\n\thash_code\x18\x04 \x01(\x04\x12\x17\n\x0fmaybe_type_name\x18\x05 \x01(\tBn\n\x18org.tensorflow.frameworkB\x0eResourceHandleP\x01Z=github.com/tensorflow/tensorflow/tensorflow/go/core/framework\xf8\x01\x01\x62\x06proto3')
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/google/protobuf/descriptor.py", line 879, in __new__
    return _message.default_pool.AddSerializedFile(serialized_pb)
TypeError: Couldn't build proto file into descriptor pool!
Invalid proto descriptor for file "tensorboard/compat/proto/resource_handle.proto":
  tensorboard.ResourceHandleProto.device: "tensorboard.ResourceHandleProto.device" is already defined in file "tensorboardX/src/resource_handle.proto".
  tensorboard.ResourceHandleProto.container: "tensorboard.ResourceHandleProto.container" is already defined in file "tensorboardX/src/resource_handle.proto".
  tensorboard.ResourceHandleProto.name: "tensorboard.ResourceHandleProto.name" is already defined in file "tensorboardX/src/resource_handle.proto".
  tensorboard.ResourceHandleProto.hash_code: "tensorboard.ResourceHandleProto.hash_code" is already defined in file "tensorboardX/src/resource_handle.proto".
  tensorboard.ResourceHandleProto.maybe_type_name: "tensorboard.ResourceHandleProto.maybe_type_name" is already defined in file "tensorboardX/src/resource_handle.proto".
  tensorboard.ResourceHandleProto: "tensorboard.ResourceHandleProto" is already defined in file "tensorboardX/src/resource_handle.proto".
```

what is the problem?? Please help me...

**Comment by batho2n** [comment]
I found the solution like this
```
$ pip uninstall protobuf
$ pip install --no-binary=protobuf protobuf
```

But, Following error message is printed

`TypeError: Conflict register for file "tensorboard/compat/proto/resource_handle.proto": tensorboard.ResourceHandleProto is already defined in file "tensorboardX/src/resource_handle.proto". Please fix the conflict by adding package name on the proto file, or use different name for the duplication.`

How can I go next step?

**Comment by batho2n** [comment]
Nextly, I do following command
```
$ pip uninstall tensorboardX
$ pip install tensorboardX
```
but, I got the error message
```
WARNING: Logging before flag parsing goes to stderr.
W0718 11:17:13.363000 139929404901120 lazy_loader.py:50]
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

FP16 Run: False
Dynamic Loss Scaling: True
Distributed Run: False
cuDNN Enabled: True
cuDNN Benchmark: False
Epoch: 0
Train loss 0 64.384430 Grad Norm 14.205294 1.74s/it
Validation loss 0: 62.002598
Traceback (most recent call last):
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/PIL/Image.py", line 2649, in fromarray
    mode, rawmode = _fromarray_typemap[typekey]
KeyError: ((1, 1, 400), '|u1')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "train.py", line 290, in <module>
    args.warm_start, args.n_gpus, args.rank, args.group_name, hparams)
  File "train.py", line 248, in train
    hparams.distributed_run, rank)
  File "train.py", line 146, in validate
    logger.log_validation(reduced_val_loss, model, y, y_pred, iteration)
  File "/home/ubuntu/users/batho2n/tacotron2/logger.py", line 34, in log_validation
    iteration)
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboardX/writer.py", line 548, in add_image
    image(tag, img_tensor, dataformats=dataformats), global_step, walltime)
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboardX/summary.py", line 216, in image
    image = make_image(tensor, rescale=rescale)
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/tensorboardX/summary.py", line 256, in make_image
    image = Image.fromarray(tensor)
  File "/home/ubuntu/miniconda3/envs/py36pt/lib/python3.6/site-packages/PIL/Image.py", line 2651, in fromarray
    raise TypeError("Cannot handle this data type")
TypeError: Cannot handle this data type
```

This is cuased that tesorboardX version is not 1.1. However, I got the fllowing error message when tenxorboardX == 1.1
```
TypeError: Conflict register for file "tensorboard/compat/proto/resource_handle.proto":
tensorboard.ResourceHandleProto is already defined in file "tensorboardX/src/resource_handle.proto". 
Please fix the conflict by adding package name on the proto file, or use different name for the duplication.
```

Please help me T_T)
